project/overview.rst

```diff
@@ -5,4 +5,5 @@ Overview
 .. toctree::
    :titlesonly:
 
-   overview/main_concepts
\ No newline at end of file
+   overview/setup
+   overview/share
```

Comment:
If this content was split in two pages, you can only define a single redirect rule. I suggest that you redirect to the main page and let the seealso guide the user to the other page if needed.